### PR TITLE
[WIP]Changes for cachito

### PR DESCRIPTION
**Pull request by jjkiely**
In order to get Porta to build using sources from Cachito, the following changes need to be merged. 
The `name` and `version` fields in the package.json are necessary to prevent a recurring error we had which asked us to add these fields.

The package.json for yarn is necessary in order to install yarn using cachito during the build process

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "name": "porta",
+  "version": "0.0.1",
```
Comment (jlledom) [inline package.json:3]: Why no setting the actual porta version `2.13.2`?

File: script/yarn/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+      "yarn": "^1.22.19"
+    }
+  }
+
```
Comment (jlledom) [inline script/yarn/package.json:16]: Better add a new line at the end of file.

**Review by jlledom — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gsaslis) [reply package.json:3]: @jlledom is that the porta version, or the 3scale version ?

Comment (gsaslis) [reply package.json:3]: (but, yes, we absolutely need to set the correct version here)

Comment (jlledom) [reply package.json:3]: @gsaslis is there a porta version different from 3scale version?

Comment (gsaslis) [reply package.json:3]: well: 

- latest 3scale product build is 2.13.4 (not 2.13.2) 
- the porta component can have a different version number than the whole product does
- we **do** use a different version for the porta (system) build - (see tags in https://catalog.redhat.com/software/containers/3scale-amp2/system-rhel7/5d80b9d7d70cc5521d516a49 )

Having said that, this here is the community porta (so a different versioning policy is 100% fine) **and** (and this is important) I don't think this `name` + `version` actually makes much difference to porta itself _anyway_ (otherwise I'd have expected to find it here already)...

tl;dr 

`42` would also work. 🙃

File: script/yarn/package.json (hunk 2 above)
Comment (akostadinov) [reply script/yarn/package.json:16]: or rather remove trainling white spaces at last line

File: package.json (hunk 1 above)
Comment (akostadinov) [reply package.json:3]: maybe instead of `0.0.1` use `placeholder` or some other value indicating that it is not supposed to correspond to actual version

Comment (jjkiely) [reply package.json:3]: The version must be in semantic versioning format according to nodejs docs, so what do you guys think would be a suitable "not a version" version? Personally, I don't see how any other version will be any better than 0.0.1, although none of them are ideal...

Comment (akostadinov) [reply package.json:3]: idk, maybe `0.0.0`, otherwise `0.0.1` looks second bet option in this case :)

Comment (jjkiely) [reply package.json]: I've set it to 0.0.0 :)

**Review by jlledom — APPROVED** [review]

**Review by akostadinov — APPROVED** [review]
